src/main/resources/templates/journals/edit.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org" xmlns:sec="http://www.thymeleaf.org/extras/spring-security">
+<head>
+    <meta charset="UTF-8"/>
+    <title>Edit a Post</title>
+    <th:block th:replace="partials/elements :: head(title='FilmSage Index')"></th:block>
+</head>
+<body>
+<nav th:replace="partials/elements :: navbar"></nav>
+<form th:action="@{/journals/{id}/edit (id = ${.getId()})}" th:method="POST" th:object="${}">
```

Comment:
@{/journals/{id}/edit (id = ${.getId()})} 
should be 
@{'/journals/' + ${journal.id} + '/edit'}

th:object="${}" 
should be 
th:object="${journal}"